src/ring_theory/power_series/well_known.lean

```diff
@@ -129,4 +129,14 @@ begin
     id_apply, rescale_one] using pow_succ' (exp A) k,
 end
 
+/-- Shows that
+$\sum_{k = 0}^{n - 1} (e^{X})^k = \sum_{p = 0}^{\infty} \sum_{k = 0}^{n - 1} \frac{k^p}{p!}X^p$. -/
+theorem exp_pow_sum [algebra ℚ A] (n : ℕ) : (finset.range n).sum(λ k, (exp A)^k) =
+power_series.mk (λ p,  (finset.range n).sum(λ k, k^p * ((algebra_map ℚ A) p.factorial⁻¹))) :=
```

Comment:
Indent two spaces when the type continues over a line.

Reply:
done